install_lib_ci.sh

```diff
@@ -4,6 +4,9 @@ wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh |
 source ~/.profile
 nvm install "$NODE_VERSION"
 while pgrep apt-get >/dev/null 2>&1 ; do
```

Comment:
Tu pourrai utiliser la commande built-in "wait" et essayer de faire un "wait pgrep apt-get >/dev/null 2>&1 ;" directement